### PR TITLE
feat: picker prototype for scale-transform on active state (SDS-12198)

### PR DESCRIPTION
**Pull request by bernhard-adobe**
https://jira.corp.adobe.com/browse/SDS-12198 Spectrum Next Picker Prototype
## Description

Bouncy Picker button or Chevron only.
Screen Shot 2022-09-26 at 15 48 25

Added a tiny bit detail in ReadME.md
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: stories/Picker/Picker.css [hunk 1]
```diff
@@ -0,0 +1,11 @@
+/*
```
Comment (bernhard-adobe) [inline stories/Picker/Picker.css:1]: I did most of my stuff in a <style>block. was quick enough.

File: stories/Picker/Picker.stories.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+      defaultValue: 80,
+      control: { type: "range", min: 50, max: 200 },
+    },
+    ease: {
+      name: "active ease function",
+      defaultValue: "ease-out",
+      options: ["ease-in-out", "ease-in", "ease-out", "ease-linear"],
+      control: { type: "select" },
+    },
+    animateChevronOnly: {
```
Comment (bernhard-adobe) [inline stories/Picker/Picker.stories.js:34]: the toggle for animating the chevron only or the entire Picker button

File: stories/Picker/Picker.js [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+  {{/if}} 
+</style>
+
+<h4>Picker scale M</h4>
+<button class="spectrum-Picker spectrum-Picker--sizeM {{#if isOpen}} is-open{{/if}}" 
+ {{#if isDisabled}} disabled{{/if}} 
+  aria-haspopup="listbox" style="width: 240px">
+  <span class="spectrum-Picker-label {{#if isPlaceholder}} is-placeholder{{/if}}">{{label}}</span>
+
+  <span class="spectrum-Picker-transform-box spectrum-Picker-menuIcon">
```
Comment (bernhard-adobe) [inline stories/Picker/Picker.js:61]: for the icon-animation, we would have to add a wrapping `<span>` as the applied transform would also transition the rotation of the chevron. It is currently not possible to transition a single transform in CSS on the same element.

https://stackoverflow.com/a/24120956

File: stories/Picker/Picker.stories.js (hunk 2 above)
Comment (GarthDB) [reply stories/Picker/Picker.stories.js:34]: nice!

File: stories/Picker/Picker.stories.js [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+      name: "only animate chevron on active state",
+      defaultValue: true,
+      type: { name: "boolean" },
+    },
+    label: { control: "text" },
+    isDisabled: {
```
Comment (GarthDB) [inline stories/Picker/Picker.stories.js:40]: Could you add the option for isQuiet?

**Review by GarthDB — CHANGES_REQUESTED** [review]
Could you add `isQuiet`, like action button? I think it could be worth testing.

File: stories/Picker/Picker.stories.js (hunk 4 above)
Comment (bernhard-adobe) [reply stories/Picker/Picker.stories.js:40]: easy enough: https://github.com/adobe/spectrum-prototypes/pull/15

**Pull request merged**